dashboard/scripts/import_course_facilitators.rb

```diff
@@ -37,29 +38,22 @@ def [](header)
 
   facilitator =
     if user_id.nil?
-      User.find_by(email: email)
+      User.find_by!(email: email)
     else
-      User.find_by(id: user_id)
+      User.find_by!(id: user_id)
     end
 
-  unless facilitator
-    STDERR.puts "Facilitator #{user_id} (#{email}) not found."
-    error_count += 1
-    next
-  end
-
   # validate other fields
   unless facilitator.name == "#{first_name} #{last_name}" && facilitator.email.casecmp(email)
```

Comment:
Make the name case insensitive? In fact, maybe go lowercase, strip out anything thats not a-z and compare that